cost/superseded_instance/superseded_instance.pt

```diff
@@ -2,9 +2,15 @@ name "Superseded Instances"
 rs_pt_ver 20180301
 type "policy"
 short_description "This Policy Template is used to automatically resize instances based on user-defined standards. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/superseded_instance) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.4"
+long_description ""
 category "Cost"
 severity "low"
+info(
+  version: "1.5",
+  provider: "RightScale",
```

Comment:
the provider should be "Flexera Optima"

Reply:
I have address the comment, please review PR